package.json

```diff
@@ -1,21 +1,20 @@
 {
   "name": "chef-workstation",
   "productName": "Chef Workstation",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "description": "Tray applicaiton for Chef Workstation.",
   "repository": "https://github.com/chef/chef-workstation-tray",
   "author": "Chef Software, Inc.",
   "main": "main.js",
   "scripts": {
     "start": "electron .",
     "build": "build --mac --win",
-    "publish": "build --mac --win -p always"
+    "release": "build --mac --win -p always"
```

Comment:
Not for this PR, but I think we should be targetting rpm/deb as well if it's supported.

Reply:
Yeah I kept those out to keep builds faster for now.